addons/binding/org.openhab.binding.amazondashbutton/README.md

```diff
@@ -0,0 +1,58 @@
+# Amazon Dash Button Binding
+
+The Amazon Dash Button is a very cheap device in order to integrate it in your home automation setup. In openHAB it can be used as a simple switch.
+
+## Prerequisites
+The binding uses pcap4j (https://www.pcap4j.org/) in order to capture ARP requests send by the Amazon Dash Button. The code is inspired by https://github.com/hortinstein/node-dash-button.
+
+
+### Install Native Library
+Let’s start with installing libpcap (for Mac/Linux/UNIX) or WinPcap (for Windows) on your computer. They are native libraries that powers the core functionalities of Pcap4J.
+#### Ubuntu
```

Comment:
Please add empty lines before and after all headlines as Jekyll will break the layout on docs.openhab.org otherwise.